modules/authentication/proc-enabling-authentication-with-microsoft-azure.adoc

```diff
@@ -33,45 +38,41 @@ To grant admin consent, a directory admin must go to the link:https://learn.micr
 
 
 .. Go to the *Certificates & Secrets* page, then the *Client secrets* tab, and create a new client secret.
-Save the *Client secret* for the next step.
-
-. Add your Microsoft Azure credentials in your {product-short} secrets.
 
-.. Edit your {product-short} secrets, such as `secrets-rhdh`.
+.. Save your **Application ID**, *Client secret*, and **Tenant ID** for the next step.
```

Comment:
It may be helpful to use the name as it appears in the Azure instance:
`Application (client) ID` and `Directory (tenant) ID`.

<img width="158" alt="image" src="https://github.com/user-attachments/assets/66b88110-57b8-4cc1-9b07-06f54d7bcccb">


Or use **Application ID** and **Directory ID** for consistency.

Reply:
Done